scm-plugins/scm-hg-plugin/src/main/java/sonia/scm/repository/spi/HgRepositoryServiceProvider.java

```diff
@@ -61,25 +57,19 @@
     Command.PULL,
     Command.MODIFY
   );
-  //J+
 
-  /** Field description */
-  public static final Set<Feature> FEATURES =
-    EnumSet.of(Feature.COMBINED_DEFAULT_BRANCH);
+  public static final Set<Feature> FEATURES = EnumSet.of(Feature.COMBINED_DEFAULT_BRANCH);
```

Comment:
Make this field `protected` like SonalCloud suggests?

Reply:
I don't know why SonarQube thinks that this should be `protected`, because `FEATURES` is used by [HgRepositoryHandler](https://github.com/scm-manager/scm-manager/blob/feature/hg_hooks_over_tcp/scm-plugins/scm-hg-plugin/src/main/java/sonia/scm/repository/HgRepositoryHandler.java#L71). Changing `FEATURES` to `protected` leads to a compiler error.